Linux integration tests are failing

https://travis-ci.com/src-d/lookout/jobs/243370464

I looks related to 
```
Failed to start rabbitmq-server.service: Unit rabbitmq-server.service not found.
error starting RabbitMQ: exit status 5
.ci/Makefile.main:261: recipe for target 'prepare-services' failed
make: *** [prepare-services] Error 1
The command "make ci-integration-dependencies" exited with 2.
```
cc @smola as it looks related to his CI tricks

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,6 +17,8 @@ addons:
     packages:
       - gcc-6
       - g++-6
+      # workaround for https://github.com/smola/ci-tricks/issues/19
+      - rabbitmq-server
 
 go:
   - '1.12.x'

